_scripts/app1-5.R

```diff
@@ -0,0 +1,22 @@
+{
+  
+  rm(list=ls()); 
+  options(show.error.locations = TRUE);
+  
+  # Prompt user to input temperature measurement location.
+ 
+  tempLocation <- readline("What is the location of your temperature measure?: "); # Enter measurement location (E.g. Yellowknife).
+  
+  # Prompt user to input temperature values, in Celsius.
+  
+  Temp1 <- as.numeric(readline("What is the value of the first temperature measurement, in Celsius?: ")); # Input first value (E.g. 24)
+  Temp2 <- as.numeric(readline("What is the value of the second measurement?: ")); # Input second value
```

Comment:
All these lines could be broken up into:
Temp1 = readline("What is the value of the second measurement?: ");
Temp1 = as.numeric(Temp1);

I am happy with wither method of doing this but, I want to point out the programming allows you to put a lot of functionality into one line of code.  And you see this a lot in R!  People cram a lot into one line in the name of efficiency* but then readability is sacrificed.  I tend to error on the side of readability because code that is readable is also code that is more reusable.  Note:  this specific line is not a case of unreadable code -- I just want to put somewhere in the back of your head some thoughts about how code can be broken down.

  * And I'd argue that you really do not gain efficiency from the perspective of the compiler